robottelo/decorators/__init__.py

```diff
@@ -199,6 +199,8 @@ class ProjectModeError(Exception):
     """Indicates an error occurred while skipping based on Project Mode."""
 
 
+# This defination is not being used currently as there are no multiple projects to run the tests on
+# Also, The setting project has been removed from properties untill wnd if we need in feature
```

Comment:
Typo "untill wnd" and also sentence not well framed

Reply:
Also, "need in feature" or "need in future"?